lib/types/multipart.js

```diff
@@ -269,34 +271,43 @@ function Multipart (boy, cfg) {
 
 Multipart.prototype.write = function (chunk, cb) {
   let r
-  if ((r = this.parser.write(chunk)) && !this._pause) { cb() } else {
+  if ((r = this.parser.write(chunk)) && !this._pause) {
+    if (this._nparts > 0 || chunk.includes(this._boundary)) {
```

Comment:
```suggestion
    if (this._nparts !== 0 || chunk.includes(this._boundary)) {
```

Imho I am a c++ guy in tis perspective: if it is never negative than make a not 0 equal check. 

Is chunk.includes even safe?  Feeld dangerous

Reply:
Sure thing, `chunk` should always be a buf, but let me check if it's really safe